app/controllers/api/web/push_subscriptions_controller.rb

```diff
@@ -17,6 +17,13 @@ def update
     render json: @push_subscription, serializer: REST::WebPushSubscriptionSerializer
   end
 
+  def destroy
+    push_subscription = ::Web::PushSubscription.find_by_token_for(:unsubscribe, params[:id])
```

Comment:
What happens if we cant find this subscription? I think it should be a no-op?

Reply:
Good point. This line will actually not fail, but the next would raise an error. I fixed that just now.